_posts/2022-04-06-#24571.md

```diff
@@ -10,15 +10,49 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+* Fingerprinting allows an attacker to recognize the same node across different
+  connections. This makes it possible to test if two addresses belong to the
+  same node, which we generally try to avoid especially for addresses belonging
+  to privacy centric networks such as Tor. Some fingerprinting attacks work
+  across restarts of the victims node, making it possible to detect if a node
+  changes addresses.
+
+* A variety of fingerprinting techniques have been previously patched. Some
+  examples include: [#9268](https://github.com/bitcoin/bitcoin/pull/9268),
+  [#8408](https://github.com/bitcoin/bitcoin/pull/8408),
+  [#23695](https://github.com/bitcoin/bitcoin/pull/23695),
+  [#11113](https://github.com/bitcoin/bitcoin/pull/11113),
+  [#11740](https://github.com/bitcoin/bitcoin/pull/11740),
+  [#5860](https://github.com/bitcoin/bitcoin/pull/5860),
+  [#5820](https://github.com/bitcoin/bitcoin/pull/5820). As shown by these PRs,
+  fingerprinting can be found in all sorts of different components such as the
+  address manager, block index or pruning logic.
+
```

Comment:
Missing a little more background on  PR #24571? Maybe a quick summary of what the fingerprintable behavior is?